sdk/spring/spring-cloud-azure-native-configuration/src/main/resources/META-INF/project.properties

```diff
@@ -0,0 +1 @@
+project.version=@project.version@
```

Comment:
why do we need this

Reply:
Only this one can be removed, I will remove it.